### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by brd**

### CODE REVIEW

File: docs/resources/vm.md [hunk 1]
```diff
@@ -29,12 +29,12 @@ EOF
...
     cpus  = 1
-    cloud_config = "${xenorchestra_cloud_config.bar.template}"
+    cloud_config = xenorchestra_cloud_config.bar.template
     name_label = "Name"
     name_description = "description"
-    template = "${data.xenorchestra_template.template.id}"
+    template = data.xenorchestra_template.template.id
     network {
-	  network_id = "${data.xenorchestra_pif.pif.network}"
+	  network_id = data.xenorchestra_pif.pif.network
```
Comment (ddelnano) [inline docs/resources/vm.md:37]: Can you switch this to a `data.xenorchestra_network` data source since that is more appropriate?

**Review by ddelnano — CHANGES_REQUESTED** [review]
Thanks for making these changes! I noticed these but put the fix in a long PR do I'm glad it's getting fixed sooner.

One minor comment otherwise looks good

**Comment by ddelnano** [comment]
@brd I went ahead and merged your change + my suggestion in #73.

**Comment by brd** [comment]
Thanks, sorry I kept getting distracted.

**Pull request closed without merging**